code/modules/reagents/chemistry_reagents/toxin.dm

```diff
@@ -254,7 +254,7 @@
 
 /datum/reagent/toxin/acid
 	name = "Sulphuric acid"
-	id = "sacid"
+	id = "sulphuric acid"
```

Comment:
code churn?

Reply:
I have to do this since it displays the id in the ui interface